ui/app/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Does this clash with the otherwise apache license of the Alertmanager? Does this have to be BSD Licensed because of dependencies or something?